bin/console

```diff
@@ -10,6 +10,14 @@ if (!is_file(dirname(__DIR__).'/vendor/autoload_runtime.php')) {
 
 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
 
+/**
+ * This file is a Symfony command-line entrypoint.
```

Comment:
How about also adding the word interface and shouldn't entrypoint be two words?

```suggestion
 * This file is a Symfony command-line interface (CLI) entry point.
```

Reply:
indeed, makes more sens to really document with full names thx